news/_posts/translation/2022-11-07-do-you-speak-prestashop-october-2022.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – October 2022 edition"
+subtitle: "Where you'll learn that PrestaShop 8 is now available"
+date: 2022-11-07
+authors: [LéaDelin]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it is also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in October!
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+### ➕ PrestaShop 8 is now available to the world 🎉
+
+PrestaShop 8 is here! While it’s been translated into multiple languages already (thank you all for your contributions!), new translation strings are added regularly, so keep an eye out. 
+Of course, other languages are still available for translation, and you are more than welcome to help contribute to the different language projects!
+
+### Proofreaders needed
+
+We are looking for proofreaders for the Spanish (Argentina), Galician and Flemish projects! So if you want to help us, reach me on either Crowdin or Slack, I’d be more than happy to guide you 🙌
```

Comment:
Careful with the "we": "we are looking for", "if you want to help us"... 
"We" could be understood as "we, the company". And the contributors are not working for the company. Be as specific as possible and rephrase the sentence to avoid using "we". Example: "Your project needs you", "Help the community", etc.

Reply:
You could also explain why proofreaders are needed. Almost fully translated but not validated, so, not available in the back office. Something like that. :)